lib/SyTest/Federation/Server.pm

```diff
@@ -373,66 +402,80 @@ sub mk_await_request_pair
       }
    };
 
+   # Don't specify the version prefix in the await function name if the version is v1 so
+   # we don't break existing function calls.
+   my ( $await_request_function_name );
+   if( $versionprefix eq "v1" ) {
+      $await_request_function_name = "await_request_$shortname";
```

Comment:
I think we should probably aim to switch to consistently using version-prefixed method names.

No problem keeping this around for now, but in addition to the prefixed name. So you end up with something like:

```perl
*{"${class}::await_request_${versionprefix}_${shortname}"} = $awaitfunc;
# deprecated alternative name for v1 endpoints
*{"${class}::await_request_${shortname}"} = $awaitfunc if ${versionprefix} eq "v1";
```